strawberry/extensions/context.py

```diff
@@ -68,7 +68,7 @@ def get_hook(self, extension: SchemaExtension) -> Optional[WrappedHook]:
                 "{self.HOOK_NAME}"
             )
         elif is_legacy:
-            warnings.warn(self.DEPRECATION_MESSAGE, DeprecationWarning)
+            warnings.warn(self.DEPRECATION_MESSAGE, DeprecationWarning, stacklevel=3)
```

Comment:
Just wanted to point out that this "only" moves the source of the warning from `strawberry.extensions.context` to `strawberry.extensions.runner` (via/ref https://github.com/blb-ventures/strawberry-django-plus/issues/180).

Full traceback:
```
Traceback (most recent call last):
  File "…/Vcs/django/django/core/handlers/exception.py", line 56, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "…/Vcs/django/django/core/handlers/base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "…/Vcs/django/django/views/generic/base.py", line 103, in view
    return self.dispatch(request, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "…/Vcs/django/django/utils/decorators.py", line 46, in _wrapper
    return bound_method(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "…/Vcs/django/django/views/decorators/csrf.py", line 55, in wrapped_view
    return view_func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "…/project/.venv/lib/python3.11/site-packages/strawberry/django/views.py", line 235, in dispatch
    result = self.schema.execute_sync(
             ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "…/project/.venv/lib/python3.11/site-packages/strawberry/schema/schema.py", line 279, in execute_sync
    result = execute_sync(
             ^^^^^^^^^^^^^
  File "…/project/.venv/lib/python3.11/site-packages/strawberry/schema/execute.py", line 176, in execute_sync
    with extensions_runner.operation():
         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "…/project/.venv/lib/python3.11/site-packages/strawberry/extensions/runner.py", line 51, in operation
    return OperationContextManager(self.extensions)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "…/project/.venv/lib/python3.11/site-packages/strawberry/extensions/context.py", line 54, in __init__
    hook = self.get_hook(extension)
           ^^^^^^^^^^^^^^^^^^^^^^^^
  File "…/project/.venv/lib/python3.11/site-packages/strawberry/extensions/context.py", line 71, in get_hook
    warnings.warn(self.DEPRECATION_MESSAGE, DeprecationWarning, stacklevel=3)
DeprecationWarning: Event driven styled extensions for on_request_start or on_request_end are deprecated, use on_operation instead
```